Cargo.toml

```diff
@@ -5,6 +5,9 @@ description = "Darklake DEX SDK - A standalone SDK for interacting directly with
 edition = "2024"
 authors = ["Darklake Team"]
 license = "MIT"
+documentation = "https://docs.rs/darklake-sdk-on-chain"
+homepage = "https://github.com/darklakefi/darklake-sdk-on-chain"
+repository = "https://github.com/darklakefi/darklake-sdk-on-chain.git"
```

Comment:
https://github.com/darklakefi/darklake-sdk-on-chain I think this should be https://github.com/darklakefi/sdk-on-chain no?